pkgs/development/python-modules/duckduckgo-search/default.nix

```diff
@@ -13,24 +13,23 @@
 
 buildPythonPackage rec {
   pname = "duckduckgo-search";
-  version = "6.3.2";
+  version = "6.3.4";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
 
-  src = fetchFromGitHub {
-    owner = "deedy5";
-    repo = "duckduckgo_search";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-5AuPAv78ePrnCr5L4CfIu/fq7Ha19zC78zg8JLu3U2A=";
+  src = fetchPypi {
+    inherit version;
+    pname = "duckduckgo_search";
+    hash = "sha256-cTF9De45PLLA+40u7cdrug2Mk8dS/pe+ADDDm4n9Bfk=";
```

Comment:
Why this?

Reply:
Because we can't run the tests anyway because they need internet access, so I figured out it could be better, but I am maybe wrong. Is fetchFromGithub preferred for fetching the sources?

Reply:
But for what I understand, PyPI package the python code so it produces an already lighter version of the package when we fetch it, it does not fetch the entire repo. Is however has the constraint that we can't run the tests because it only package the python module files, but in our case this doesn't have an impact because we don't run them.

Reply:
`duckduckgo-search` src method does not have a direct impact, but we can revert it if you prefer

Reply:
I would revert it yes, there's no need change it unless there's a good reason.

> But for what I understand, PyPI package the python code so it produces an already lighter version of the package when we fetch it, it does not fetch the entire repo.

It is possible, I don't know

Using Github is the insurance that the complete repo is fetched, including the tests.

Reply:
Amended it to [python3Packages.duckduckgo-search: 6.3.2 -> 6.3.4](https://github.com/NixOS/nixpkgs/pull/355393/commits/731c5123ac05e935fe9042cde61f842d39d12712) commit